mergekit/merge.py

```diff
@@ -229,6 +242,36 @@ def _copy_tokenizer(
     tokenizer.save_pretrained(out_path, safe_serialization=True)
 
 
+def _copy_feature_extractor(
+    merge_config: MergeConfiguration, out_path: str, trust_remote_code: bool = False
+):
+    donor_model = merge_config.base_model or (merge_config.referenced_models()[0])
+
+    if (os.path.exists(
+            os.path.join(donor_model.model.path, "preprocessor_config.json")
+        )
+    ):
+        logging.info(f"Copying feature_extractor from {donor_model}")
+
+        for file_name in [
+            "preprocessor_config.json",
+        ]:
+            if os.path.exists(os.path.join(donor_model.model.path, file_name)):
+                shutil.copy(
+                    os.path.join(donor_model.model.path, file_name),
+                    os.path.join(out_path, file_name),
+                )
+        return
+
+    # fallback: try actually loading the feature_extractor and saving it
+    logging.info(f"Reserializing feature_extractor from {donor_model}")
+    feature_extractor = transformers.AutoFeatureExtractor.from_pretrained(
+        donor_model.model.path,
+        revision=donor_model.model.revision,
+        trust_remote_code=trust_remote_code,
+    )
+    _set_chat_template(feature_extractor, merge_config)
```

Comment:
I think this is incorrect - feature extractors don't have a chat template do they? Not that it seems to be breaking anything but it's extra work anyway.